.github/workflows/daily.yml

```diff
@@ -261,7 +261,7 @@ jobs:
         repository: ${{ env.GITHUB_REPOSITORY }}
         ref: ${{ env.GITHUB_HEAD_REF }}
     - name: make
-      run: make valgrind REDIS_CFLAGS='-Werror -DREDIS_TEST'
+      run: make valgrind REDIS_CFLAGS='-Werror -DREDIS_TEST' LUA_DEBUG=yes
```

Comment:
are we sure it's a good idea to add that to the valgrind run (already kinda slow).
is valgrind needed in order to detect these errors? i suppose it isn't.

Reply:
We do not need valgrind for this but I thought its might be a good place because this is where we catch such issues no? Where else do you suggest to put it?

Reply:
i'm not sure.
the options are:
1. add it to `test-ubuntu-jemalloc`
2. create a new section, which will only run scripting.tc. and maybe the module tests?
3. add it to `test-sanitizer-address` or `test-sanitizer-undefined`

thing is that valgrind is very slow, and this could make it slower (causing more false positive timing failures).
the address sanitizer is also slow in some cases, but not as as slow as valgrind.
the UB sanitizer is not as slow AFAIK.

p.s. since it's not directly related to any of them, if we piggyback one of them, maybe we better add a comment (saying it's just a piggyback)

Reply:
I believe that if we go with approach 2 we should still run all the tests, I see tests that performs `eval` all over.